src/components/PokemonSprite.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import PokeSprite from 'react-poke-sprites';
+import 'react-tippy/dist/tippy.css';
+import { Tooltip } from 'react-tippy';
+
+const containerStyle = {
+  textAlign: 'left',
+  width: 300,
+};
+
+const colStyle = {
+  float: 'left',
+  width: '33%',
+};
+
+const rowStyle = {
+  overflow: 'auto',
+  height: '100%',
+};
+
+const Stats = ({ baseStats }) => (
+  <React.Fragment>
+    <div> Stats: </div>
+    <div style={rowStyle}>
+      <div style={colStyle}>Hp: {baseStats.hp} </div>
```

Comment:
For some reason I couldn't seem to get regular className css imports to be applied to the tooltip. I ended up going with this route until I could figure out why. 

I imported the css above.
```import ../css/PokemonSprite.css```

Then I used it in these div tags but none of the styles were coming through. Upon inspecting the developer tools, under the styling tab in firefox, I verified that I could view my css. I am sure I am missing something extremely silly. If anyone can spot what I did wrong, please let me know.

Thanks

Reply:
nvm, I think it was caching related. Got it working.